File: Modul-06/SOAL 1/PRAK601-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int BARIS, KOLOM;

    printf("Input: \n");
    scanf("%d %d", &BARIS, &KOLOM);

    int MATRIKS[BARIS][KOLOM];

    for (int i = 0; i < BARIS; i++) {
        for (int b = 0; b < KOLOM; b++) {
            scanf("%d", &MATRIKS[i][b]);
        }
    }
    printf("\nOutput: \n");
    for (int i = 0; i < BARIS; i++) {
        for (int b = 0; b < KOLOM; b++) {
            printf("%d ", MATRIKS[i][b]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul-06/SOAL 1/PRAK601-2310817210009-MuhammadIbnuSina.py
print("Input: ")
def BENTUK_MATRIKS(BARIS, KOLOM, AV):
    MATRIKS = [[0] * KOLOM for _ in range(BARIS)]
    index = 0

    for i in range(BARIS):
        for b in range(KOLOM):
            MATRIKS[i][b] = AV[index]
            index += 1

    return MATRIKS

def main():
    BARIS, KOLOM = map(int, input().split())

    ANGKA_MATRIKS = list(map(int, input().split()))

    HASIL_MATRIKS = BENTUK_MATRIKS(BARIS, KOLOM, ANGKA_MATRIKS)

    print("\nOutput: ")
    for i in range(BARIS):
        for b in range(KOLOM):
            print(HASIL_MATRIKS[i][b], end=" ")
        print()

if __name__ == "__main__":
    main()
